quedex_api/user_stream.py

```diff
@@ -234,6 +250,16 @@ def cancel_order(self, cancel_order_command):
     else:
       self._encrypt_send(cancel_order_command)
 
+  def cancel_all_orders(self):
+    if not self._initialized:
+      raise Exception('UserStream not initialized, wait until UserStreamListener.on_read is called')
+    cancel_all_orders_command = dict(type='cancel_all_orders')
```

Comment:
Not very pythonic way of creating a dict...

Reply:
And the pythonic is { key: value } ?

Reply:
Yes.